pkgs/by-name/pa/pam_rundir/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  pam,
+}:
+stdenv.mkDerivation rec {
+  pname = "pam_rundir";
+  version = "1.0.0";
+
+  src = fetchurl {
+    url = "https://jjacky.com/pam_rundir/pam_rundir-${version}.tar.gz";
+    sha256 = "sha256-x3m2me0jd3o726h7f2ftOV/pV/PJYTj67kX4eie8wCA=";
+  };
+
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace-fail /usr/share/man /share/man
+  '';
+
+  buildInputs = [
+    pam
+  ];
+
+  configureFlags = [
+    "--securedir=/lib/security"
+    "--with-parentdir=/run/user"
+  ];
+
+  installFlags = [
+    "DESTDIR=$(out)"
+  ];
+
```

Comment:
add nix-update-script

Reply:
is that actually needed? i thought the scripts/bots did a decent job of taking care of this without the attribute?

Reply:
anyways, it occurs to me that these scripts won't even work given this is a random download